frontend/.storybook/preview.tsx

```diff
@@ -20,7 +21,19 @@ dayjsUtils.init()
 
 const withReactQuery: DecoratorFn = (storyFn) => {
   const queryClient = new QueryClient({
-    defaultOptions: { queries: { staleTime: Infinity } },
+    defaultOptions: {
```

Comment:
would it make sense to have a constant for the query client config in the app, and import and reuse it here? to make sure storybook react-query behaviour matches the app

Reply:
there are actually some differences between the config in storybook and the actual app (we actually do not want them to be the same!)

for example, the staleness should be `infinite` in storybook. now that i'm looking at it, i should also set to zero retries in storybook (done in 3d432ab)